crates/bevy_ui/src/render/mod.rs

```diff
@@ -532,16 +562,39 @@ const UI_CAMERA_FAR: f32 = 1000.0;
 // TODO: Evaluate if we still need this.
 const UI_CAMERA_TRANSFORM_OFFSET: f32 = -0.1;
 
+/// The ID of the subview associated with a camera on which UI is to be drawn.
+///
+/// When UI is present, 3D cameras extract to two views: a 3D one and a UI one.
+/// The main 3D camera gets subview 0, and the corresponding UI camera gets this
+/// subview, 1.
```

Comment:
Why does this comment only mentions 3d cameras? Isn't this also true for 2d?

Reply:
Yes every active 2d or 3d camera gets an extracted UI view. Doesn't matter if there are any UI entities present, only that the UI rendering plugin is enabled.

Reply:
Updated the comment.